lib/plugins/expressionParser.js

```diff
@@ -159,6 +159,17 @@ function applyExpression(expression, context, typeInformation) {
 
```

Comment:
Documentation at https://github.com/telefonicaid/iotagent-node-lib/blob/master/doc/expressionLanguage.md#ngsiv2-support should be changed consecuently.

Reply:
Fixed in b91aba6773d975e3e73d7cb61c4f499001b06e2f